src/app/renderer.js

```diff
@@ -91,6 +91,13 @@ Renderer.prototype.contracts = function (data, source) {
     })
   }
 
+  var retrieveMetadataHash = function (bytecode) {
+    var match = /a165627a7a72305820([0-9a-f]{64})0029$/.exec(bytecode)
```

Comment:
I think we should rather just lowercase the bytecode so the output is consistent.